ibtk/src/lagrangian/FEDataManager.cpp

```diff
@@ -518,7 +514,8 @@ FEDataManager::reinitElementMappings()
     // We reinitialize mappings after repartitioning, so clear the cache since
     // its content is no longer relevant:
     d_fe_data->clearCached();
-    d_L2_proj_matrix_diag.clear();
+
+    // The ghosted diagonal mass matrix needs to be rebuilt after repartitioning.
     d_L2_proj_matrix_diag_ghost.clear();
```

Comment:
We shouldn't have this and the other calls any more since it is no longer this class' responsibility to manage the solvers.

Reply:
@drwells notes this below, but the IB ghosted representation should be owned by `FEDataManager` or possibly `FEData`.